ui/src/pages/project.tsx

```diff
@@ -105,68 +45,33 @@ export const Project = ({
     );
   }
 
-  // we must render the tab contents outside of the Antd tabs component to prevent layout issues in the ProjectDetails component
-  const renderTab = (key: string) => {
-    switch (key) {
-      case 'pipelines':
-        return (
-          <Pipelines
-            project={data?.result?.value as _Project}
-            creatingStage={creatingStage}
-            creatingWarehouse={creatingWarehouse}
-          />
-        );
-      case 'credentials':
-        return config?.secretManagementEnabled ? (
-          <CredentialsList />
-        ) : (
-          <Pipelines project={data?.result?.value as _Project} />
-        );
-      case 'analysisTemplates':
-        return <AnalysisTemplatesList />;
-      case 'events':
-        return <Events />;
-      case 'roles':
-        return <Roles />;
-      case 'promotionTasks':
-        return <PromotionTasks />;
-      default:
-        return <Pipelines project={data?.result?.value as _Project} />;
-    }
-  };
-
   return (
     <div className='h-full flex flex-col'>
-      <div className='px-6 pt-5 pb-3 mb-2'>
-        <div className='flex items-center'>
-          <div className='mr-auto'>
-            <SmallLabel>PROJECT</SmallLabel>
-            <div className='text-2xl font-semibold flex items-center'>
-              {name} <ProjectSettings />
-            </div>
-            <Description
-              loading={isLoading}
-              item={data?.result?.value as _Project}
-              className='mt-1'
-            />
-          </div>
-          <div className='flex items-center gap-8 text-gray-500 text-sm mr-2'>
-            {Object.entries(tabs).map(([key, value]) => (
-              <Tooltip key={key} title={value.label}>
-                <div
-                  className={classNames('cursor-pointer', { 'text-blue-500': tab === key })}
-                  onClick={() => {
-                    navigate(generatePath(tabs[key as keyof typeof tabs]?.path ?? '', { name }));
-                  }}
-                >
-                  <FontAwesomeIcon icon={value.icon} className='mr-2' />
-                </div>
-              </Tooltip>
-            ))}
-          </div>
-        </div>
-      </div>
-      {renderTab(tab)}
+      <BaseHeader>
+        <Breadcrumb separator='>' items={[projectBreadcrumbs[0], { title: name }]} />
+        <Space>
+          <Button
+            icon={<FontAwesomeIcon icon={faClockRotateLeft} size='sm' />}
+            onClick={() => navigate(generatePath(paths.projectEvents, { name }))}
+            size='small'
+          >
+            Events
+          </Button>
+          <Button
+            icon={<FontAwesomeIcon icon={faCog} />}
+            onClick={() => navigate(generatePath(paths.projectSettings, { name }))}
+            size='small'
+          >
+            Settings
+          </Button>
+        </Space>
+      </BaseHeader>
```

Comment:
Can we have this header outside of normal loading state?

Reply:
Yes, I wanted to have it, but it needs more changes to avoid a double loader.

![image](https://github.com/user-attachments/assets/458751f5-74d9-4ef1-964c-f2b92a272535)